app/code/Magento/AsynchronousOperations/Ui/Component/Operation/DataProvider.php

```diff
@@ -68,7 +68,7 @@ private function getSummaryReport($operationDetails)
 
         $summaryReport = __('%1 items selected for mass update', $operationDetails['operations_total'])->__toString();
         if ($operationDetails['operations_successful'] > 0) {
-            $summaryReport .= __(', %1 successfully updated', $operationDetails['operations_successful']);
+            $summaryReport .= __(', %1 successfully updated', $operationDetails['operations_total']);
```

Comment:
What the reason to change this key to total ? there actually exist failed test for this behavior.
![DeepinScreenshot_select-area_20200206111900](https://user-images.githubusercontent.com/51681547/73923012-dad5d400-48d2-11ea-98df-9ee1d10c5e30.png)